docs/data_access.rst

```diff
@@ -8,6 +8,11 @@ PUDL data, so if you have a suggestion please `open a GitHub issue
 <https://github.com/catalyst-cooperative/pudl/issues>`__. If you have a question you
 can `create a GitHub discussion <https://github.com/orgs/catalyst-cooperative/discussions/new?category=help-me>`__.
 
+PUDL's primary data output is the ``pudl.sqlite`` database. We recommend working
```

Comment:
Maybe explain that this is where the cleaned data lives? And maybe a caveat about CEMS?